packages/db/db-client-types.ts

```diff
@@ -155,7 +158,7 @@ export type ContentItemOutput = {
   cid: definitions['content']['cid']
   dagSize?: definitions['content']['dag_size']
   pins: Array<PinItemOutput>,
-  deals: Array<Deal>
+  deals?: Array<Deal>
```

Comment:
Can you please add some rationale behind this change? Just struggling to follow the whole chain, it'd really help if you could add some context.

Reply:
Ok reinstated, since it is a required attribute for  `getStatus`